assets/js/components/GoogleChart.js

```diff
@@ -179,7 +193,7 @@ export default function GoogleChart( props ) {
 
 GoogleChart.propTypes = {
 	chartID: PropTypes.string,
-	chartType: PropTypes.oneOf( [ 'pie', 'line', '' ] ),
+	chartType: PropTypes.oneOf( [ 'area', 'pie', 'line' ] ).isRequired,
```

Comment:
Why make this prop required? We could easily keep the default of `line`, this way the PR also doesn't affect too many other areas this late in the release cycle.

Reply:
Mostly that I found the default of "line" quite confusing and it'd add to the switch-case… we could do this later but I tested every spot this is in and it didn't fail any VRTs either, so is safe to make I think.

The default of `line` didn't seem very intuitive, especially with multiple options for this prop now. We can revert this change, but I think the original prop setup wasn't very intentional.

If you'd still prefer we can revert this though, let me know 😄

Reply:
Fair point, I was mostly wanting to avoid unrelated changes this late in the cycle. But this one should be straightforward enough.